### PR TITLE
Add SQL Scheduler docs

### PR DESCRIPTION
**Pull request by matkuliak**
## What's Inside
Documentation for the new feature "SQL Job Scheduler"

## Preview
https://crate-cloud--59.org.readthedocs.build/en/59/reference/overview.html#failed-cloning
(just below. why i'm linking to this explained below)

## Highlights
- Introduction of the feature
- UI navigation to/around the feature. Explanation on the "Overview" and "Logs" tabs.
- Examples
  - Cleanup job - removes entries older than 30 days, runs once a day
  - Copying to another table job - copies from system table into user created table - runs every hour
  - Didn't find any other good examples, feel free to send any my way if 2 is not enough or they're not good enough
- Limitations and possible issues

- ~~PR also contains few unrelated link fixes, leftovers from https://github.com/crate/cloud-docs/pull/55 that were not caught before. Also deletion of `sign-up.rst`, it complained about bunch of stuff and we don't need it anymore so deletion was easier.~~

## Checklist

 - [x] Link to issue this PR refers to (if applicable): https://github.com/crate/cloud/issues/1658
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
Found one issue when I was going to link the header "SQL Scheduler". The consequence of [this fix](https://github.com/crate/cloud-docs/pull/55#discussion_r1510795056) is that the header written this way can't be linked to. I will look for alternative, that should stop progress of this PR.

File: docs/_assets/img/cluster-sql-scheduler-example-copying.png
Comment (amotl) [inline docs/_assets/img/cluster-sql-scheduler-example-copying.png:1]: All the images are way too large, no?

File: docs/tutorials/sign-up.rst
Comment (amotl) [inline docs/tutorials/sign-up.rst:1]: I don't think deleting this file, and corresponding refactorings, should go into the context of this PR, let alone into the same commit, even if its about "docs only", for the sake of separation of concerns.

File: docs/tutorials/deploy/stripe.rst [hunk 1]
```diff
@@ -138,4 +138,4 @@ Your cluster should now be ready to use!
 .. _Cloud Console: https://console.cratedb.cloud/
 .. _region: https://crate.io/docs/cloud/reference/en/latest/glossary.html#region
 .. _Stripe: https://stripe.com
-.. _subscription plan: https://crate.io/docs/cloud/reference/en/latest/subscription-plans.html
\ No newline at end of file
+.. _subscription plan: https://cratedb.com/docs/cloud/en/latest/reference/services.html#
```
Comment (amotl) [inline docs/tutorials/deploy/stripe.rst:141]:
```suggestion
.. _subscription plan: https://cratedb.com/docs/cloud/en/latest/reference/services.html
```

File: docs/tutorials/deploy/stripe.rst [hunk 2]
```diff
@@ -59,8 +59,8 @@ by selecting the "Request new region" button and filling out the form.
 .. image:: ../../_assets/img/deployment-region-request.png
    :alt: CrateDB Cloud Console Deployment New Region Request
 
-If you are deploying an :ref:`Edge <edge>` cluster, you can choose your
-custom region by selecting the "Add a custom edge region" link.
+If you are deploying an :ref:`Edge <cloud-on-kubernetes>` cluster, you can
+choose your custom region by selecting the "Add a custom edge region" link.
```
Comment (amotl) [inline docs/tutorials/deploy/stripe.rst:63]: Fixing broken links should be a separate commit, specifically when the main one is about a dedicated feature, like in this case. When having maintenance PRs, or other sweeping PRs mixing concerns, sure enough it is a different story.

Comment (matkuliak) [reply docs/tutorials/deploy/stripe.rst:63]: That is fair, I'll move them that to its own thing.

File: docs/_assets/img/cluster-sql-scheduler-example-copying.png
Comment (matkuliak) [reply docs/_assets/img/cluster-sql-scheduler-example-copying.png:1]: You're right, Oxipng didn't run, fixed.

**Comment by matkuliak** [comment]
Build will fail, ignore for now, https://github.com/crate/cloud-docs/pull/60 needs to be merged first.

**Review by proddata — APPROVED** [review]
Thanks 🙏

File: docs/_assets/img/cluster-sql-scheduler-example-copying.png
Comment (amotl) [inline docs/_assets/img/cluster-sql-scheduler-example-copying.png:1]: That file still weighs in with +507 KB. Is it legit?

File: docs/_assets/img/cluster-sql-scheduler-example-cleanup.png
Comment (amotl) [inline docs/_assets/img/cluster-sql-scheduler-example-cleanup.png:1]: +326 KB for that one. Acceptable?

Comment (matkuliak) [reply docs/_assets/img/cluster-sql-scheduler-example-cleanup.png:1]: Used _oxipng_ on all of them 🤷🏻‍♂️
From looking at other screenshots they're about average, no? Not sure I can do much more.

Comment (amotl) [reply docs/_assets/img/cluster-sql-scheduler-example-cleanup.png:1]: I just wonder why that one is much larger than the others.

Comment (amotl) [reply docs/_assets/img/cluster-sql-scheduler-example-cleanup.png:1]: [docs/_assets/img/cluster-sql-console.png](https://github.com/crate/cloud-docs/pull/66/files#diff-1b010b81b76c0a4d07f6c9a2e8300af13702575e2854062d2677e3f6340b867b) from GH-66 is also pretty large, but I also can't get it smaller, even after reducing its physical width.

File: docs/_assets/img/cluster-sql-scheduler-example-copying.png
Comment (amotl) [reply docs/_assets/img/cluster-sql-scheduler-example-copying.png:1]: > From looking at other screenshots they're about average, no? Not sure I can do much more.

[docs/_assets/img/cluster-sql-console.png](https://github.com/crate/cloud-docs/pull/66/files#diff-1b010b81b76c0a4d07f6c9a2e8300af13702575e2854062d2677e3f6340b867b) from GH-66 is also pretty large, but I also can't get it smaller, even after reducing its physical width.

So, I figure it will be good to go. Thanks.

**Review by amotl — APPROVED** [review]

File: docs/_assets/img/cluster-sql-scheduler-example-copying.png
Comment (matkuliak) [reply docs/_assets/img/cluster-sql-scheduler-example-copying.png:1]: Okay, thought I was doing something wrong for a sec. But they are somewhat big, it's weird. 
I will look into it a bit more. Thank you.
